### ISSUE TITLE
Migrate content from MediWiki bot page on wiki to user manual

### ISSUE THREAD

**Issue by ostephens** [OpenRefine/OpenRefine#3575]
New user manual has details about creating a bot for Wikidata https://docs.openrefine.org/manual/wikidata#manage-wikidata-account

There is a page on the wiki that gives more general information about creating a bot account for any Wikibase instance https://github.com/OpenRefine/OpenRefine/wiki/MediaWiki-Bot-Passwords

The more general information should be migrated across to the new user manual and then the MediaWiki-Bot-Passwords page in the wiki can be deprecated (replace the content with a link to the relevant page in the new user manual)

**Comment by AhwiniMirajkar** [comment]
I want to work on this

**Comment by MohamedMetwalli5** [comment]
Could I work on this issue?

**Comment by wetneb** [comment]
@MohamedMetwalli5 sure! @AhwiniMirajkar sorry for the lck of reaction on this…

**Comment by Shatakshi0805** [comment]
Can I work on this?

**Comment by Shatakshi0805** [comment]
I would like to work on this issue.

**Comment by wetneb** [comment]
@Shatakshi0805 sure, thank you!

**Comment by Shatakshi0805** [comment]
Thank you for assigning. I needed a little help as I cloned the wiki repo on my machine I'm not able to find the Mediawiki Bot passwords page in the OpenRefine.wiki folder that I Cloned and I need it to edit and use its content. I am able to see other files But I am not able to locate the file which I need to edit and copy the content from. Could you please help me with the same?

**Comment by thadguidry** [comment]
@Shatakshi0805 you should see it now.  Try to clone again.  Looks like we had an error on an older FAQ page title that was causing a checkout issue.  Let me know if it is now resolved and that you can see the file needed when checked out the `master` branch as I just did to verify it works now.

![image](https://user-images.githubusercontent.com/986438/122949008-f2372500-d340-11eb-9522-372b8b352a54.png)

**Comment by wetneb** [comment]
Otherwise you could simply copy the source of the page at https://github.com/OpenRefine/OpenRefine/wiki/MediaWiki-Bot-Passwords/_edit

**Comment by Shatakshi0805** [comment]
Yes, It works. I am now able to see the files by checking on the master branch and then cloning. Thank you so much @thadguidry and @wetneb for helping. Really appreciate it! :)

**Comment by Shatakshi0805** [comment]
![image](https://user-images.githubusercontent.com/78600829/123029997-ac8d6180-d3ff-11eb-8821-38a6d6367dc6.png)

Apologies for sharing image like this and not the actual website image with the applied changes. I was not able to see the changes on the website that I made after `yarn start` so had to share this image. Just want to know if the changes made are correct in the overview.md file. I am currently a beginner in GitHub and have not made contributions to such a large project so sometimes get a little overwhelmed if I not make wrong changes.

**Comment by Shatakshi0805** [comment]
I had one last question. Apologies for asking questions so many times as I am a beginner and facing these issues. [Q]  After making changes in docs and saving them and running successfully 
on local host through `yarn start` I am not able to see the changes that I made on the local host server. Is it like this or I have missed some step in this process?

**Comment by thadguidry** [comment]
yarn should have opened your local browser and reflected the changes very quickly in seconds on the browser window that it popped up... without having to restart the server.  See https://github.com/OpenRefine/OpenRefine/tree/master/docs#local-development

Why don't you hop onto our Gitter chat and we can troubleshoot live.  You can use the Element client to reach it easily here: https://app.element.io/#/room/#OpenRefine_OpenRefine:gitter.im

**Comment by Shatakshi0805** [comment]
Yes, I saved the file and then ran on local server. And the only issue I am facing is not able to see the changes that I made on the website. Do let me know if there is something that I can help with.

**Comment by thadguidry** [comment]
@Shatakshi0805 You can help by finding out why it's doing that?  Ask the docusaurus or yarn community?  Because I also have the same problem now and have no idea and tried a few things for over an hour at that time.

**Comment by wetneb** [comment]
@Shatakshi0805 beware that we have versioning in place: perhaps you are editing the dev version of a page, but you are looking at the released version (3.4) of the page on the website?

You could also try making some random changes to `docs/versioned_docs/version-3.4/index.md` and check if you can see them at `http://localhost:3000`.

**Comment by thadguidry** [comment]
@wetneb Yeap, that works correctly now!

So it seems that the version "current" really means whats in the /docs/docs folder from [what I read ](https://docusaurus.io/docs/versioning#figure-out-the-behavior-for-the-current-version)
and not the versioned docs under `versioned_docs\version-3.4`

`versions.json` :
```json
[
  "current"
]
```

For us that is current docs I.E. for 3.5 version...not older 3.4 version?

So to edit and see the docs for older 3.4 version, we just need to change to

`versions.json` :
```json
[
  "3.4"
]
```

In a separate issue, I'm not sure about the `docusaurus.config.js` change what we might want to do for
`lastVersion: undefined` for the doc plugin section itself  
```
        /**
         * The last version is the one we navigate to in priority on versioned sites
         * It is the one displayed by default in docs navbar items
         * By default, the last version is the first one to appear in versions.json
         * By default, the last version is at the "root" (docs have path=/docs/myDoc)
         * Note: it is possible to configure the path and label of the last version
         * Tip: using lastVersion: 'current' make sense in many cases
         */
        lastVersion: undefined,
```

**Comment by thadguidry** [comment]
@Shatakshi0805 Checkout our `master` branch again and notice the updated `README.md` file under `/docs/docs` which should help you on this issue now.

**Comment by Shatakshi0805** [comment]
Thank you Thad for guidance. I will try this now and will update.

**Comment by thadguidry** [comment]
@Shatakshi0805 No problem, and if I didn't explain it well enough in that `README.md` then feel free to submit a PR to improve that also!

**Comment by Shatakshi0805** [comment]
yes, it seems clear to me.

**Comment by khadijm** [comment]
hi @wetneb  can i continue to work on this issues? thanks

**Comment by khadijm** [comment]
if possible please can you assign this issue to me thanks

**Comment by khadijm** [comment]
ready for review @ostephens  and @wetneb   https://github.com/OpenRefine/OpenRefine/pull/4652
